File: fix.py
with open("src/components/EmployeeBooking.tsx", "r") as f:
    text = f.read()

target = """            </div>
          )}
            </div>
          </div>
      </div>

      {/* Create New Room Modal */}"""

replacement = """            </div>
          )}
            </div>
          </div>
        </div>
      )}

      {/* Create New Room Modal */}"""

with open("src/components/EmployeeBooking.tsx", "w") as f:
    f.write(text.replace(target, replacement))

File: fix.js
import fs from 'fs';
let text = fs.readFileSync('src/components/EmployeeBooking.tsx', 'utf8');

const target = `            </div>
          )}
            </div>
          </div>
      </div>

      {/* Create New Room Modal */}`;

const replacement = `            </div>
          )}
            </div>
          </div>
        </div>
      )}

      {/* Create New Room Modal */}`;

fs.writeFileSync('src/components/EmployeeBooking.tsx', text.replace(target, replacement));
